src/client/modules/counter/containers/Counter.spec.js

```diff
@@ -7,7 +7,7 @@ import COUNTER_SUBSCRIPTION from '../graphql/CounterSubscription.graphql';
 chai.should();
 
 const COUNTER_APOLLO_VALUE = 11;
-const COUNTER_REDUX_VALUE = 7;
+const COUNTER_REDUX_VALUE = 1;
```

Comment:
Why it is named REDUX here and why you change its value at all? Every change should have a reason.